test-utils/runtime/src/lib.rs

```diff
@@ -114,6 +114,8 @@ pub enum Extrinsic {
 	ChangesTrieConfigUpdate(Option<ChangesTrieConfiguration>),
 }
 
+parity_util_mem::malloc_size_of_is_0!(Extrinsic); // non-opaque extrinisic does not need this
```

Comment:
Why don't we use this macro for the other ones where we return `0`?

Reply:
Indeed, I thought it does not support generics, but it now does, thanks!

Reply:
Looks like it is not properly supported though
And it also adds bounds on generic arguments which are not required in this case, so I won't use `malloc_size_of_is_0!` for now, but I'll add another macro later.

Reply:
`malloc_size_of_is_0!` is upgraded, and one case covered